packages/editor-ui/src/utils/nodeViewUtils.ts

```diff
@@ -741,6 +741,19 @@ export const moveBackInputLabelPosition = (targetEndpoint: Endpoint) => {
 	}
 };
 
+export const addConnectionTestData = (connectionInfo: ConnectionEstablishedParams) => {
```

Comment:
We can make this more DRY be slightly refactoring this method and using it both in NodeView and `addConnectionActionsOverlay`
```
export const addConnectionTestData = (
	source: HTMLElement,
	target: HTMLElement,
	el: HTMLElement,
) => {
	const sourceNodeName = source.getAttribute('data-name')?.toString();
	const targetNodeName = target.getAttribute('data-name')?.toString();

	if (el && sourceNodeName && targetNodeName) {
		el.setAttribute('data-source-node', sourceNodeName);
		el.setAttribute('data-target-node', targetNodeName);
	}
};
```

And then in `addConnectionActionsOverlay` it's just `addConnectionTestData(component.source, component.target, div);` 
And in NodeView: 
```
if ('canvas' in info.connection.connector) {
							setTimeout(() => {
								NodeViewUtils.addConnectionTestData(
									info.source,
									info.target,
									info.connection.connector.canvas as HTMLElement,
								);
							}, 0);
						}
```